packages/webpack-config-single-spa-react/lib/webpack-config-single-spa-react.js

```diff
@@ -3,6 +3,7 @@ const webpackConfigSingleSpa = require("webpack-config-single-spa");
 module.exports = webpackConfigSingleSpaReact;
 
 function webpackConfigSingleSpaReact(opts) {
+  opts.react = true;
```

Comment:
Just curious, what's this do?

Reply:
https://github.com/single-spa/create-single-spa/pull/62/files#diff-9629c0751be0b0a313fe13ae8b46351eR29